catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/model/Search.java

```diff
@@ -102,6 +102,8 @@
 
     public static final String CACHED = "cached";
 
+    private static final String INTERNAL_LOCAL_RESOURCE = "internal.local-resource";
```

Comment:
After a discussion with @pklinef, @brendan-hofmann, and @lessarderic yesterday, we decided to write a ticket to address this issue.  The ticket has been added to the backlog.